docs/contributor-guide/python.md

```diff
@@ -1,6 +1,8 @@
-# Python
+# Contributing
```

Comment:
The header levels on this page is a little funky. It seems like the organization should be:

- GitHub Merge Requests
- Pre-Commit
- Python
    - PEP 8
    - Black
    - Type Annotations and MyPy
    - Raising Exceptions
    - Cryptography
- Docstrings

...?
This would involved adding the "Python" section back in. Whatever you decide, the hierarchy should make sense -- having "PEP 8" under "Pre-commit" as is currently is confusing.